plugin-flex-ts-template-v2/src/feature-library/enhanced-crm-container/flex-hooks/events/pluginsInitialized.ts

```diff
@@ -0,0 +1,10 @@
+import * as Flex from '@twilio/flex-ui';
+
+import { FlexEvent } from '../../../../types/feature-loader';
+
+export const eventName = FlexEvent.pluginsInitialized;
+export const eventHook = function registerAction(flex: typeof Flex, _manager: Flex.Manager) {
+  flex.Actions.registerAction('CRMContainerLoaded', async () => {
```

Comment:
i realize why you've called it CRMContainerLoaded but I wonder if something like LoadCRMContainerTabs would be more appropriate and intuitive since the way we are using it (if im following correctly) is to use the before hook to capture them all when invoking the event

Reply:
maybe thats not the best name either, but something thats a little more intuitive

Reply:
Ha, I struggled with this name too 😄 I am good with `LoadCRMContainerTabs`!